wget no longer supports --passive-ftp flag: Unknown option 'passive-ftp'

MKDIR scripts
DOWNLOAD scripts/config.sub
Unknown option 'passive-ftp'
gmake: *** [~/koxtoolchain/build/CT_NG_BUILD/share/crosstool-ng/scripts/scripts.mk:51: scripts/config.sub] Error 2

--- a/scripts/scripts.mk
+++ b/scripts/scripts.mk
@@ -32,7 +32,7 @@ ifneq ($(strip $(V)),2)
 endif
 
 ifneq ($(CT_WGET),)
-download_cmd = $(CT_WGET) --passive-ftp $(wget_silent_opt) -O $@
+download_cmd = $(CT_WGET) $(wget_silent_opt) -O $@
 else
 ifneq ($(CT_CURL),)
 download_cmd = $(CT_CURL) --ftp-pasv $(curl_silent_opt) -o $@

